### ISSUE TITLE
Compatibility table

### ISSUE THREAD

**Issue by axeldelafosse** [nissy-dev/swc-plugin-react-native-web#32]
Hey @nissy-dev,

Thank you for your work!

We worked with Software Mansion to port the Reanimated plugin to SWC and quickly realized that the compatibility with new versions of SWC / Next.js was pretty hard to follow.

We found that having a compatibility table in the README would help users quickly check which version of the plugin is compatible. Cf this README: https://www.npmjs.com/package/react-native-reanimated-swc-plugin?activeTab=readme. It doesn't prevent errors if using mismatched versions but at least give some kind of guidance to people looking for a solution.

Would you consider adding a similar table to your README?

Thanks,
Axel

**Comment by nissy-dev** [comment]
Thank you for your advice! I added the compatibility table in #33.
